lib/osf-components/addon/components/registries/schema-block-renderer/label/label-content/styles.scss

```diff
@@ -1,5 +1,10 @@
 .Required {
     color: $brand-danger;
+    margin-left: 5px;
+}
+
+.LabelWrapper {
+    white-space: pre-wrap;
```

Comment:
I like to try to keep styles roughly in the order they are used (makes this one come first).